modules/cms-kit/src/Volo.CmsKit.Admin.Application/Volo/CmsKit/Admin/CmsKitAdminApplicationModule.cs

```diff
@@ -44,11 +46,35 @@ private void ConfigureTagOptions()
                         new TagEntityTypeDefiniton(
                             BlogPostConsts.EntityType,
                             LocalizableString.Create<CmsKitResource>("BlogPost"),
-                            CmsKitAdminPermissions.BlogPosts.Update,
-                            CmsKitAdminPermissions.BlogPosts.Update,
-                            CmsKitAdminPermissions.BlogPosts.Update));
+                            new[] { CmsKitAdminPermissions.BlogPosts.Create, CmsKitAdminPermissions.BlogPosts.Update },
```

Comment:
Why are these three lines the same?

Reply:
All 3 actions are dependent on BlogPost.Create & BlogPost.Update for tags.